M2/Macaulay2/packages/Varieties/backports.m2

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
What changes are in this file?

Reply:
Ah sorry this was used during the workshop, should have been removed in https://github.com/Macaulay2/M2/commit/d90df86a02f09eb22644644af8db8fa932949579. I'll do it in the next push.